comp/core/workloadmeta/types.go

```diff
@@ -77,6 +77,10 @@ const (
 	// SourceRemoteProcessCollector reprents processes entities detected
 	// by the RemoteProcessCollector.
 	SourceRemoteProcessCollector Source = "remote_process_collector"
+
+	// SourceLanguageDetectionComponent represents container languages
+	// detected by node agents
+	SourceLanguageDetectionNodeAgent Source = "language_detection_node_agent"
```

Comment:
Perhaps removing `NodeAgent` could be pertinent here. I am suggesting this for two reasons:
- With respect to the DCA, the information comes from the LanguageDetectionServer (or Handler). Whether the source is the node agent might not be so important.
- Having `NodeAgent` in the name is a bit counter intuitive. For someone who discovers the code for the first time, the source is not obvious. Here the source is the pld server.


Also the name doesn't match the name of the variable (`SourceLanguageDetectionNodeAgent`) in the comment (`SourceLanguageDetectionComponent`)